### ISSUE TITLE
Updated to JDK 20

### ISSUE THREAD

**Issue by N0usy** [TormStorm/jagex-launcher-linux#25]
Hi, I updated to JDK 20, and I am now not able to download it. Does the java version have anything to do with this?

**Comment by nmlynch94** [comment]
Not able to download what? Are you trying to reinstall the Jagex Launcher after a jdk update? Please clarify what you are trying to do and what errors you are seeing.

**Comment by N0usy** [comment]
I had the launcher downloaded . Updated to JDK 20 from 11. After this when I tried to launch the Jagex Launcher I got "contenstatid" error. I then uninstalled the launcher and tried to reinstall. Now I'm met with "wine: RLIMIT_NICE is <= 20, unable to use setpriority safely". The installer launches and when I try to close the installer with the terimnal it won't work, so I just used system monitor to end the task. Went to bed and now it works, I still get the RLIMIT_NICE from wine, but was able to download.
